exporter/tanzuobservabilityexporter/README.md

```diff
@@ -61,6 +61,11 @@ reduce memory
 consumption. [Details and defaults here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md)
 .
 
+**NOTE:** The order matters when enabling multiple processors in a pipeline (e.g. the memory limiter and batch processors in the
+example config below). Please refer to the
```

Comment:
I'd suggest keeping sentences all on one line rather than breaking them onto multiple lines. It may help prevent rendering issues like the one above.

Reply:
Fixed.